### PR TITLE
Updates in May 2019

### PR DESCRIPTION
**Pull request by phil-opp**
This PR adds an "Updates in May 2019" post that gives an overview of the recent changes to the blog and the tools/libraries behind the scenes.

### CODE REVIEW

File: blog/content/status-update/2019-06-03.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
++++
+title = "Updates in Mai 2019"
```
Comment (joerixaop) [inline blog/content/status-update/2019-06-03.md]: spelling: this should be "Updates in May 2019"

Comment (bjorn3) [reply blog/content/status-update/2019-06-03.md]: We noticed it at the same time :)

**Comment by phil-opp** [comment]
@joerixaop @bjorn3 Thanks a lot! My muscle memory automatically writes the German "Mai" :D. Should be fixed now.

**Comment by alex-bender** [comment]
@phil-opp take a look at this tool: https://github.com/client9/misspell
It's doing great job fixing those damn typos.
And thanks for content -- it's really great!

**Comment by phil-opp** [comment]
Looks useful, thanks!

**Pull request merged**